src/frontend/src/lib/components/ui/ExternalLink.svelte

```diff
@@ -20,6 +20,8 @@
 	class:text-primary={color === 'blue'}
 	class:hover:text-inherit={color === 'blue'}
 	class:active:text-inherit={color === 'blue'}
+	class:hover:text-blue-ribbon={color === 'inherit'}
```

Comment:
it's a bit weird no? Blue is inherit but inherit is text-blue-ribbon.

Should we rename the value 'inherit'?

Reply:
well, those are the `hover`/`active` classes right? The idea is that `blue`/`inherit` refers to the main color of the text in the normal state

Reply:
I would honestly expect `class:hover:text-inherit={color === 'inherit'}`

Reply:
Ah now I understand. Yeah it's a bit confusing.

Reply:
The color is the idle state and the class is the active state.